src/benchmark.cr

```diff
@@ -134,8 +134,6 @@ module Benchmark
   # Benchmark.realtime { "a" * 100_000 } # => 00:00:00.0005840
   # ```
```

Comment:
Shouldn't this method be removed or deprecated if it's just an alias to `Time.measure`?

Reply:
I would support either removing `Time.measure` or moving entire `Benchmark` somewhere else.

Reply:
What does `moving entire Benchmark somewhere else` mean?

Reply:
Removing `Benchmark` module but keeping its functionality somewhere.

Reply:
Ah. Well but that's out of the scope of this PR ;)

Reply:
Yes, either `Benchmark.realtime` or `Time.measure` should be dropped. Since `Time.measure` is always available and has a broader meaning (doesn't imply benchmarking), I would remove `Benchmark.realtime` but I don't have a strong opinion.

Removing Benchmark altogether is out-of-scope, indeed.

Reply:
I'd say `Time.measure` is definitely better: This feature is not specificly benchmarking, just measuring time. Thus it's also better understandable than `Benchmark.realtime`.

Reply:
The problem I see is that people looking at `Benchmark` module will be overwhelmed because there's not a single method returning a directly usable amount of time. In case of this removal, please add a reference to `Time.measure` to `Benchmark`.